README.md

```diff
@@ -1,4 +1,5 @@
 <p align="center">
+<!-- TODO: this image does not exist -->
```

Comment:
Was this removed in a PR? Regardless, we need an image, probably one with a dark background.